res/skins/Shade/style.qss

```diff
@@ -281,6 +281,9 @@ QHeaderView {
   font-size: 10pt;
   font-weight: bold;
   color: #0f0f0f;
+  border-right: 1px solid #0f0f0f;
```

Comment:
small hint for a small fix: we don't need the border here ;)

Reply:
done